build.sbt

```diff
@@ -1117,6 +1118,59 @@ lazy val searcher = project
   .dependsOn(testkit % Test)
   .dependsOn(`polyglot-api`)
 
+lazy val `polyglot-ydoc-server` = project
+  .in(file("lib/java/polyglot-ydoc-server"))
+  .enablePlugins(JPMSPlugin)
+  .configs(Test)
+  .settings(
+    crossPaths := false,
+    autoScalaLibrary := false,
+    Compile / run / fork := true,
+    Test / fork := true,
+    modulePath := {
+      JPMSUtils.filterModulesFromUpdate(
+        update.value,
+        GraalVM.modules ++ Seq(
+          "io.helidon.builder"       % "helidon-builder-api"         % helidonVersion,
```

Comment:
This list is a proof that we should_invest in discovering transitive deps_ in `sbt` and compute them automatically.

Reply:
Right. This is a known limitation of the `JPMSPlugin`. `modulePath` setting is a `Seq[File]`.

Reply:
In `bazel` we will have to list those explicitly always, same as here.